platform/default/default_file_source.cpp

```diff
@@ -19,6 +20,12 @@ bool isAssetURL(const std::string& url) {
     return std::equal(assetProtocol.begin(), assetProtocol.end(), url.begin());
 }
 
+const std::string fileProtocol = "file://";
```

Comment:
Better avoid this. See for alternatives: https://github.com/mapbox/mapbox-gl-native/pull/6455